app/src/main/java/com/pluu/webtoon/data/impl/AbstractWeekApi.kt

```diff
@@ -0,0 +1,29 @@
+package com.pluu.webtoon.data.impl
+
+import com.pluu.webtoon.data.WeeklyRequest
+import com.pluu.webtoon.di.NetworkUseCase
+import com.pluu.webtoon.item.Result
+import com.pluu.webtoon.item.ToonInfo
+import java.util.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/e6d7b5dbb11d2d4fb917c63fbb0187d6361e5aea/app/src/main/java/com/pluu/webtoon/data/impl/AbstractWeekApi.kt#L7"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>